tests/pass/extern_types.rs

```diff
@@ -1,5 +1,6 @@
 //@revisions: stack tree
 //@[tree]compile-flags: -Zmiri-tree-borrows
+//@[stack]only-miri: miri specific warning
```

Comment:
```suggestion
//@only-miri: miri specific warning
```

Reply:
But the test works, it just differs with the diagnostics